File: TwitterKitIOS/TimelinesObjC/TimelinesObjC/ViewController.m
//
//  ViewController.m
//  TimelinesObjC
//
//  Created by Twitter on 5/26/15.
//  Copyright (c) 2015 Twitter. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    TWTRAPIClient *client = [[TWTRAPIClient alloc] init];
    self.dataSource = [[TWTRUserTimelineDataSource alloc] initWithScreenName:@"twitterdev" APIClient:client];
}

@end

File: TwitterKitIOS/TimelinesSwift/TimelinesSwift/ViewController.swift
import UIKit
import TwitterKit

class ViewController: TWTRTimelineViewController {
    
    override func viewDidLoad() {
        super.viewDidLoad()

        //The screen name allows to get all tweets for that account in the timeline
        //self.dataSource = TWTRUserTimelineDataSource(screenName: "ASCETweets", apiClient: TWTRAPIClient())
        //Using seach query for hashtags
        
        let client = TWTRAPIClient()
        self.dataSource = TWTRSearchTimelineDataSource(searchQuery: "#ios", apiClient: client)
        
        }
        
    
    
}
